pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -96,6 +96,7 @@ stdenv.mkDerivation {
   name = targetPrefix
     + (if name != "" then name else stdenv.lib.removePrefix targetPrefix "${ccName}-wrapper")
     + (stdenv.lib.optionalString (cc != null && ccVersion != "") "-${ccVersion}");
+  version = ccVersion;
```

Comment:
On the line above we conditionally append `"-${ccVersion}"` suffix and here you refer to `ccVersion` unconditionally. I wonder what is the use case for cc-wrapper with `cc = null`, depending on what it is we might not want to break it (`ccVersion` tries to evaluate `cc.name`).

Reply:
`cc == null` is for this native stdenv stuff I think we should be deprecating anyways.

Reply:
OK, I've added a null check. We still have the minor issue that a null cc will result in a null version attribute rather than a missing one, which seems to be against convention. I don't know an elegant way to conditionally add an attribute to the derivation...

Reply:
Great! Yeah this fine for now / master.